WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -7,10 +7,13 @@ import protocol WooFoundation.Analytics
 protocol PointOfSaleAggregateModelProtocol {
     var orderStage: PointOfSaleOrderStage { get }
 
+    var cardReaderConnectionStatus: CardPresentPaymentReaderConnectionStatus { get }
+    func connectCardReader()
+    func disconnectCardReader()
```

Comment:
Nice, I see you're grouping them by _subject_ inside the protocol, I'm guessing a next step is to split this into smaller interfaces where each encapsulates its own card/item-list/cart related responsibilities?

Reply:
Yep, where possible. I think there will be an interface surface in the posModel itself, but I'm hopeful that can be a state property and some proxied function calls which go to another service which would handle the logic.